source/stylesheets/pages/_l_atelier.scss

```diff
@@ -0,0 +1,41 @@
+#atelier-pictures {
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  height: 400px;
+
+  img {
+    height: 400px
+  }
+}
+
+#dots {
+  width: 100%;
+  height: 400px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+
+  .dot {
+    margin: 16px;
+    height: 10px;
+    width: 10px;
+    cursor: pointer;
+  }
+}
+
```

Comment:
```
.dot {
   margin: 16px;
   height: 10px;
   width: 10px;
   cursor: pointer;
}
```

Est redondant avec .dot-lg
#dots > .dot => size 10px
.dot-lg => size 12px

Est ce qu'on uniformiserait pas les deux ? En rajoutant la nouvelle classe .empty dans > dots.scss :

```
.dot-lg {
[...]
 &.empty {
    background: transparent;
    border: 1px solid $green;
  }
} 

```
Pour la partie #dots, on peut aussi la bouger dans dots.scss. Up to you :smile: